scripts/make_docker_image/booking/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM openjdk:8-jre-alpine
+
+RUN mkdir -p /opt/tank/
+
+COPY ./acmeair-booking-service-2.0.0-SNAPSHOT-exec.jar  /opt/tank
+
+
+CMD ["java", "-jar","-Dspring.profiles.active=cse","-Dspring.data.mongodb.database=acmeair","-Dspring.data.mongodb.host=mongodb","-Dspring.data.mongodb.port=32701","/opt/tank/acmeair-booking-service-2.0.0-SNAPSHOT-exec.jar"]
```

Comment:
Please check the readme file for the name of exec jar, and take the maven docker plugin setting as an example.